### PR TITLE
Create animal-sniffer_ubuntu_20.04.sh

### PR DESCRIPTION
**Pull request by nageshlop**
To create the build script for package and verify the specific jar file is generated after the build on local vm.

### CODE REVIEW

File: a/animal-sniffer/animal-sniffer_ubuntu_20.04.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+#Install Updates
+sudo apt-get clean
+sudo apt-get update
+sudo apt-get install maven
+
+#Build and test animal-sniffer package
+if [ $# -lt 1 ]
+then
+  echo "Usage: $0 branch or tag_name"
```
Comment (gururajrkatti) [inline a/animal-sniffer/animal-sniffer_ubuntu_20.04.sh:29]: Will it accept both branch name and tag name ? Also, it should build a default version if we don't pass any argument, as per the requirement. Pl. also upload the license file.

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by nageshlop** [comment]
Test results with tag as argument.

ubuntu@nagesh-at:/scratch/jar-proj$ ./test-a-sniffer.sh animal-sniffer-parent-1.20
Cloning the target
Cloning into 'animal-sniffer'...
remote: Enumerating objects: 395, done.
remote: Counting objects: 100% (395/395), done.
remote: Compressing objects: 100% (213/213), done.
remote: Total 395 (delta 106), reused 298 (delta 47), pack-reused 0
Receiving objects: 100% (395/395), 700.97 KiB | 3.75 MiB/s, done.
Resolving deltas: 100% (106/106), done.
Note: switching to 'a69b280569ff52694cb2cec65b907b85ee41c517'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Apache Maven 3.6.3
Maven home: /usr/share/maven
Java version: 11.0.10, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-ppc64el
Default locale: en, platform encoding: UTF-8
OS name: "linux", version: "5.4.0-42-generic", arch: "ppc64le", family: "unix"
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] Animal Sniffer                                                     [pom]
[INFO] Java boot classpath detector                                       [jar]
[INFO] Animal Sniffer Annotations                                         [jar]
[INFO] Animal Sniffer                                                     [jar]
[INFO] Animal Sniffer Maven Plugin                               [maven-plugin]
[INFO] Animal Sniffer Maven Enforcer Rule                                 [jar]
[INFO] Animal Sniffer Ant Tasks                                           [jar]
[INFO]
[INFO] --------------< org.codehaus.mojo:animal-sniffer-parent >---------------
[INFO] Building Animal Sniffer 1.20                                       [1/7]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ animal-sniffer-parent ---
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ animal-sniffer-parent ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/pom.xml to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-parent/1.20/animal-sniffer-parent-1.20.pom
[INFO]
[INFO] -----------< org.codehaus.mojo:java-boot-classpath-detector >-----------
[INFO] Building Java boot classpath detector 1.20                         [2/7]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ java-boot-classpath-detector ---
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ java-boot-classpath-detector ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/java-boot-classpath-detector/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ java-boot-classpath-detector ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /scratch/jar-proj/animal-sniffer/java-boot-classpath-detector/target/classes
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ java-boot-classpath-detector ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/java-boot-classpath-detector/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ java-boot-classpath-detector ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ java-boot-classpath-detector ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:3.2.0:jar (default-jar) @ java-boot-classpath-detector ---
[INFO] Building jar: /scratch/jar-proj/animal-sniffer/java-boot-classpath-detector/target/java-boot-classpath-detector-1.20.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ java-boot-classpath-detector ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/java-boot-classpath-detector/target/java-boot-classpath-detector-1.20.jar to /root/.m2/repository/org/codehaus/mojo/java-boot-classpath-detector/1.20/java-boot-classpath-detector-1.20.jar
[INFO] Installing /scratch/jar-proj/animal-sniffer/java-boot-classpath-detector/pom.xml to /root/.m2/repository/org/codehaus/mojo/java-boot-classpath-detector/1.20/java-boot-classpath-detector-1.20.pom
[INFO]
[INFO] ------------< org.codehaus.mojo:animal-sniffer-annotations >------------
[INFO] Building Animal Sniffer Annotations 1.20                           [3/7]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ animal-sniffer-annotations ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-java) @ animal-sniffer-annotations ---
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ animal-sniffer-annotations ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer-annotations/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ animal-sniffer-annotations ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /scratch/jar-proj/animal-sniffer/animal-sniffer-annotations/target/classes
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ animal-sniffer-annotations ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer-annotations/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ animal-sniffer-annotations ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ animal-sniffer-annotations ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:3.2.0:jar (default-jar) @ animal-sniffer-annotations ---
[INFO] Building jar: /scratch/jar-proj/animal-sniffer/animal-sniffer-annotations/target/animal-sniffer-annotations-1.20.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ animal-sniffer-annotations ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-annotations/target/animal-sniffer-annotations-1.20.jar to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-annotations/1.20/animal-sniffer-annotations-1.20.jar
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-annotations/pom.xml to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-annotations/1.20/animal-sniffer-annotations-1.20.pom
[INFO]
[INFO] ------------------< org.codehaus.mojo:animal-sniffer >------------------
[INFO] Building Animal Sniffer 1.20                                       [4/7]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ animal-sniffer ---
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ animal-sniffer ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ animal-sniffer ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 10 source files to /scratch/jar-proj/animal-sniffer/animal-sniffer/target/classes
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ animal-sniffer ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 1 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ animal-sniffer ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /scratch/jar-proj/animal-sniffer/animal-sniffer/target/test-classes
[INFO]
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ animal-sniffer ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:3.2.0:jar (default-jar) @ animal-sniffer ---
[INFO] Building jar: /scratch/jar-proj/animal-sniffer/animal-sniffer/target/animal-sniffer-1.20.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ animal-sniffer ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer/target/animal-sniffer-1.20.jar to /root/.m2/repository/org/codehaus/mojo/animal-sniffer/1.20/animal-sniffer-1.20.jar
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer/pom.xml to /root/.m2/repository/org/codehaus/mojo/animal-sniffer/1.20/animal-sniffer-1.20.pom
[INFO]
[INFO] -----------< org.codehaus.mojo:animal-sniffer-maven-plugin >------------
[INFO] Building Animal Sniffer Maven Plugin 1.20                          [5/7]
[INFO] ----------------------------[ maven-plugin ]----------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ animal-sniffer-maven-plugin ---
[INFO]
[INFO] --- maven-plugin-plugin:3.6.0:helpmojo (help-mojo) @ animal-sniffer-maven-plugin ---
[INFO] Using 'UTF-8' encoding to read mojo source files.
[INFO] java-javadoc mojo extractor found 0 mojo descriptor.
[INFO] java-annotations mojo extractor found 0 mojo descriptor.
[INFO]
[INFO] --- maven-plugin-plugin:3.6.0:helpmojo (help-goal) @ animal-sniffer-maven-plugin ---
[INFO] Using 'UTF-8' encoding to read mojo source files.
[INFO] java-javadoc mojo extractor found 0 mojo descriptor.
[INFO] java-annotations mojo extractor found 0 mojo descriptor.
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ animal-sniffer-maven-plugin ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 1 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ animal-sniffer-maven-plugin ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 7 source files to /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/target/classes
[INFO] /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/src/main/java/org/codehaus/mojo/animal_sniffer/maven/CheckSignatureMojo.java: Some input files use or override a deprecated API.
[INFO] /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/src/main/java/org/codehaus/mojo/animal_sniffer/maven/CheckSignatureMojo.java: Recompile with -Xlint:deprecation for details.
[INFO] /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/src/main/java/org/codehaus/mojo/animal_sniffer/maven/CheckSignatureMojo.java: Some input files use unchecked or unsafe operations.
[INFO] /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/src/main/java/org/codehaus/mojo/animal_sniffer/maven/CheckSignatureMojo.java: Recompile with -Xlint:unchecked for details.
[INFO]
[INFO] --- maven-plugin-plugin:3.6.0:descriptor (default-descriptor) @ animal-sniffer-maven-plugin ---
[INFO] Using 'UTF-8' encoding to read mojo source files.
[INFO] java-javadoc mojo extractor found 0 mojo descriptor.
[INFO] java-annotations mojo extractor found 3 mojo descriptors.
[INFO]
[INFO] --- maven-plugin-plugin:3.6.0:descriptor (mojo-descriptor) @ animal-sniffer-maven-plugin ---
[INFO] Using 'UTF-8' encoding to read mojo source files.
[INFO] java-javadoc mojo extractor found 0 mojo descriptor.
[INFO] java-annotations mojo extractor found 3 mojo descriptors.
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ animal-sniffer-maven-plugin ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ animal-sniffer-maven-plugin ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ animal-sniffer-maven-plugin ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:3.2.0:jar (default-jar) @ animal-sniffer-maven-plugin ---
[INFO] Building jar: /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/target/animal-sniffer-maven-plugin-1.20.jar
[INFO]
[INFO] --- maven-plugin-plugin:3.6.0:addPluginArtifactMetadata (default-addPluginArtifactMetadata) @ animal-sniffer-maven-plugin ---
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ animal-sniffer-maven-plugin ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/target/animal-sniffer-maven-plugin-1.20.jar to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-maven-plugin/1.20/animal-sniffer-maven-plugin-1.20.jar
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-maven-plugin/pom.xml to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-maven-plugin/1.20/animal-sniffer-maven-plugin-1.20.pom
[INFO]
[INFO] -----------< org.codehaus.mojo:animal-sniffer-enforcer-rule >-----------
[INFO] Building Animal Sniffer Maven Enforcer Rule 1.20                   [6/7]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ animal-sniffer-enforcer-rule ---
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ animal-sniffer-enforcer-rule ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ animal-sniffer-enforcer-rule ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 3 source files to /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/target/classes
[INFO] /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/src/main/java/org/codehaus/mojo/animal_sniffer/enforcer/CheckSignatureRule.java: /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/src/main/java/org/codehaus/mojo/animal_sniffer/enforcer/CheckSignatureRule.java uses unchecked or unsafe operations.
[INFO] /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/src/main/java/org/codehaus/mojo/animal_sniffer/enforcer/CheckSignatureRule.java: Recompile with -Xlint:unchecked for details.
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ animal-sniffer-enforcer-rule ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ animal-sniffer-enforcer-rule ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ animal-sniffer-enforcer-rule ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:3.2.0:jar (default-jar) @ animal-sniffer-enforcer-rule ---
[INFO] Building jar: /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/target/animal-sniffer-enforcer-rule-1.20.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ animal-sniffer-enforcer-rule ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/target/animal-sniffer-enforcer-rule-1.20.jar to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-enforcer-rule/1.20/animal-sniffer-enforcer-rule-1.20.jar
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-enforcer-rule/pom.xml to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-enforcer-rule/1.20/animal-sniffer-enforcer-rule-1.20.pom
[INFO]
[INFO] -------------< org.codehaus.mojo:animal-sniffer-ant-tasks >-------------
[INFO] Building Animal Sniffer Ant Tasks 1.20                             [7/7]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (mojo-enforcer-rules) @ animal-sniffer-ant-tasks ---
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ animal-sniffer-ant-tasks ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 1 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ animal-sniffer-ant-tasks ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 6 source files to /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/target/classes
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ animal-sniffer-ant-tasks ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ animal-sniffer-ant-tasks ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ animal-sniffer-ant-tasks ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:3.2.0:jar (default-jar) @ animal-sniffer-ant-tasks ---
[INFO] Building jar: /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/target/animal-sniffer-ant-tasks-1.20.jar
[INFO]
[INFO] --- maven-shade-plugin:3.2.4:shade (default) @ animal-sniffer-ant-tasks ---
[INFO] Including org.codehaus.mojo:animal-sniffer:jar:1.20 in the shaded jar.
[INFO] Including org.ow2.asm:asm:jar:9.1 in the shaded jar.
[WARNING] Discovered module-info.class. Shading will break its strong encapsulation.
[WARNING] animal-sniffer-1.20.jar, animal-sniffer-ant-tasks-1.20.jar, asm-9.1.jar define 1 overlapping resource:
[WARNING]   - META-INF/MANIFEST.MF
[WARNING] maven-shade-plugin has detected that some class files are
[WARNING] present in two or more JARs. When this happens, only one
[WARNING] single version of the class is copied to the uber jar.
[WARNING] Usually this is not harmful and you can skip these warnings,
[WARNING] otherwise try to manually exclude artifacts based on
[WARNING] mvn dependency:tree -Ddetail=true and the above output.
[WARNING] See http://maven.apache.org/plugins/maven-shade-plugin/
[INFO] Replacing original artifact with shaded artifact.
[INFO] Replacing /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/target/animal-sniffer-ant-tasks-1.20.jar with /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/target/animal-sniffer-ant-tasks-1.20-shaded.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ animal-sniffer-ant-tasks ---
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/target/animal-sniffer-ant-tasks-1.20.jar to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-ant-tasks/1.20/animal-sniffer-ant-tasks-1.20.jar
[INFO] Installing /scratch/jar-proj/animal-sniffer/animal-sniffer-ant-tasks/pom.xml to /root/.m2/repository/org/codehaus/mojo/animal-sniffer-ant-tasks/1.20/animal-sniffer-ant-tasks-1.20.pom
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Animal Sniffer 1.20:
[INFO]
[INFO] Animal Sniffer ..................................... SUCCESS [  0.793 s]
[INFO] Java boot classpath detector ....................... SUCCESS [  1.636 s]
[INFO] Animal Sniffer Annotations ......................... SUCCESS [  0.320 s]
[INFO] Animal Sniffer ..................................... SUCCESS [  1.028 s]
[INFO] Animal Sniffer Maven Plugin ........................ SUCCESS [  3.282 s]
[INFO] Animal Sniffer Maven Enforcer Rule ................. SUCCESS [  0.459 s]
[INFO] Animal Sniffer Ant Tasks ........................... SUCCESS [  0.699 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.527 s
[INFO] Finished at: 2021-04-19T04:30:25Z
[INFO] ------------------------------------------------------------------------
ubuntu@nagesh-at:/scratch/jar-proj$

**Comment by gururajrkatti** [comment]
Hi @nageshlop pl. put the apache 2.0 license. It is the license for all the build scripts.

**Comment by nageshlop** [comment]
@gururajrkatti updated.

**Review by gururajrkatti — COMMENTED** [review]
LGTM. Pl. squash the commits

File: a/animal-sniffer/animal-sniffer_ubuntu_20.04.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+else
+        echo "Cloning the target"
+fi
+
+# Build and test the package
+# BRANCHES can be master, or any tags available ex : animal-sniffer-parent-1.20, animal-sniffer-parent-1.19 etc.
+
+git clone --depth 1 --branch $BRANCH $URL --single-branch
+cd $PKG_NAME
+sudo mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```
Comment (seth-priya) [inline a/animal-sniffer/animal-sniffer_ubuntu_20.04.sh:52]: if there are no test files why do we need to set -DskipTests=true?

Comment (nageshlop) [reply a/animal-sniffer/animal-sniffer_ubuntu_20.04.sh:52]: Will update.

**Comment by seth-priya** [comment]
@nageshlop please squash the commits

**Pull request merged**